Drop ARM softfp bot

aoikonomopoulos opened the pull request:
#### a5dc4a30262019be32a9623622ebac6e40eb78c6
<pre>
Drop ARM softfp bot
<a href="https://bugs.webkit.org/show_bug.cgi?id=242236">https://bugs.webkit.org/show_bug.cgi?id=242236</a>

Reviewed by Aakash Jain.

The softfp ARMv7 bot has been kept around to ensure that things would
work for users that are depend on vendor libraries that require the
softfp ABI. Recent changes broke softfp and we&apos;re no longer aware of any
users for it. Bug 242172 drops JSC support for the softfp ABI; this bug
is about removing the corresponding buildbot from build.webkit.org.

* Tools/CISupport/build-webkit-org/config.json:
* Tools/CISupport/build-webkit-org/factories_unittest.py:
(TestExpectedBuildSteps):
* Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/WebKitBuildbot.js:
(WebKitBuildbot):

Canonical link: <a href="https://commits.webkit.org/252115@main">https://commits.webkit.org/252115@main</a>
</pre>

aperezdc reviewed (changes requested):
Ahhh, I didn't notice this one and made almost the same changes in #1991 

I do notice that this version is missing the edit in `Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/WebKitBuildbot.js`, which I expect we want.

aperezdc reviewed (commented):
LGTM, you may want to get a final r+ from @aj062 or @JonWBedard

aj062 commented on Tools/CISupport/build-webkit-org/config.json:
```diff
@@ -561,11 +561,6 @@
-                    {
-                      "name": "JSCOnly-Linux-ARMv7-Thumb2-SoftFP-Release", "factory": "BuildAndJSCTestsFactory",
-                      "platform": "jsc-only", "configuration": "release", "architectures": ["armv7"],
-                      "workernames": ["jsconly-linux-igalia-bot-5"]
```
Do you want to delete "jsconly-linux-igalia-bot-5" bot from this config.json, or assign it to some other queue or just leave it as is for now?

aoikonomopoulos replied:
Oops. I guess it's best that we just delete it and ask for a new worker if we ever need an extra one. Updating the PR shortly.

aoikonomopoulos committed:
Drop ARM softfp bot
https://bugs.webkit.org/show_bug.cgi?id=242236

Reviewed by Aakash Jain.

The softfp ARMv7 bot has been kept around to ensure that things would
work for users that are depend on vendor libraries that require the
softfp ABI. Recent changes broke softfp and we're no longer aware of any
users for it. Bug 242172 drops JSC support for the softfp ABI; this bug
is about removing the corresponding buildbot from build.webkit.org.

* Tools/CISupport/build-webkit-org/config.json:
* Tools/CISupport/build-webkit-org/factories_unittest.py:
(TestExpectedBuildSteps):
* Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/WebKitBuildbot.js:
(WebKitBuildbot):

Canonical link: https://commits.webkit.org/252115@main
